### ISSUE TITLE
[doc] Error in index.rst

### ISSUE THREAD

**Issue by javiereguiluz** [doctrine/DoctrineFixturesBundle#163]
As reported in http://symfony.com/doc/build_errors:

```
Errors for "doctrine-fixtures-bundle/master"

index.rst:4: WARNING: unknown document: /book/doctrine
index.rst:4: WARNING: unknown document: /bundles/DoctrineMongoDBBundle/index
```

**Comment by stof** [comment]
I think this should use external links because they are in separate parts of the symfony.com documentation, right ?

**Comment by javiereguiluz** [comment]
You are right. Fixed it in #164 by using external links.
